.goreleaser.yaml

```diff
@@ -0,0 +1,27 @@
+# Make sure to check the documentation at https://goreleaser.com
+
+builds:
+  - id: cloudcost-explorer
+    binary: cloudcost-explorer
+    main: ./cmd/exporter/exporter.go
+  - env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+    goarch:
+     - amd64
+     - arm64
+
+archives:
+  - format: tar.gz
+    # this name template makes the OS and Arch compatible with the results of uname.
+    name_template: >-
+      {{ .ProjectName }}_
+      {{- title .Os }}_
+      {{- if eq .Arch "amd64" }}x86_64
+      {{- else }}{{ .Arch }}{{ end }}
+      {{- if .Arm }}v{{ .Arm }}{{ end }}
+checksum:
+  name_template: 'checksums.txt'
+changelog:
+  sort: asc
```

Comment:
@Pokom we can add some rules here to exclude `chore` or `docs` if you want. My thinking was that we could leave all PR titles for now and iterate later if we want to make the release notes leaner.

Reply:
Agreed on `docs` and think that would be fine to add here, assuming it's a simple check. `chore` may be related to code changes, so I would hold off on excluding `chore` PR's

Reply:
added exclusions for docs and specifically chore(deps) :)